views/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+
+    <link rel="stylesheet" href="/style.css" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Anton|Libre+Franklin&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+    <h1><a href="#"  class="logo">Search engine</a></h1>
+    <form action="/search" method="post">
+        <input type="text" name="keyword" id="keyword" placeholder="Search ____ or type a URL" />
+        <fieldset>
+          <input type="submit" name="searchButton" id="searchButton" value="Submit" />
+          <input type="button" name="randomButton" id="randomButton" value="Random" />
```

Comment:
May be useful to change input type to button for the randomButton item as well? You will want similar functionality (getting a URL on submit), or you will run into similar issues as with the submit button.